Makefile

```diff
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+PROJECT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
```

Comment:
We have another PROJECT_DIR below, can we reuse?

Reply:
Good call! I think I copyied that one but forgot to remove old one.

Reply:
Well I learned that make file scoping is weird.  I moved it up because I would think we should only use a variable after declaring it.  But I guess that isn't necessary in Makefiles.

I just use the same variable now.